### PR TITLE
Move log out to the footer menu for now

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

File: app/js/views/navigation-menu.jsx [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
+    this.foxbox = props.foxbox;
+  }
+
   shouldComponentUpdate() {
     // We never need to update this component as it is being recreated each time
     // the route changes.
     return false;
   }
 
+  handleOnClick(evt) {
```
Comment (azasypkin) [inline app/js/views/navigation-menu.jsx]: __nit:__ ```evt``` is not used.

File: app/js/views/navigation-menu.jsx [hunk 2]
```diff
@@ -1,19 +1,29 @@
...
+    this.foxbox.logout();
+
+    // Once logged out, we redirect to the login page.
+    location.hash = '#users/login';
```
Comment (azasypkin) [inline app/js/views/navigation-menu.jsx]: __nit:__ we can likely just set this hash directly on the link itself, in theory event handler is called first and only then default action is performed.

**Comment by azasypkin** [comment]
r+ with nits addressed and green Travis

**Comment by gmarty** [comment]
Nits addressed and tests fixed.

**Pull request merged**